bson-record-codec/src/test/unit/org/bson/codecs/record/samples/TestRecord.java

```diff
@@ -23,6 +23,7 @@
 
 import java.util.List;
 
+@SuppressWarnings("deprecation")
 public record TestRecord(String name,
```

Comment:
Suggest rename to `TestRecordWithDeprecatedAnnotations`

Reply:
Good idea.  Done.  Also replaced most usage of it with `TestRecordWithPojoAnnotations`